scripts/index.js

```diff
@@ -1,146 +1,226 @@
+
+    kookaburra: {
+        name: "Kookaburra",
+        lifespan: "20 years",
+        group: "Birds",
+        image: "images/transparent-img/kookaburra.png",
+        food: "Insects and small animals including snakes, frogs and lizards",
+        shortDescription:   `Native to the eucalyptus forests of eastern Australia, the laughing kookaburra is the largest 
+                            member of the Kingfisher family, with females weighing up to one pound and growing to 43 cm 
+                            in length.`,
+        fulldescription:    `Its beak can reach 10 cm long and is used to snatch a variety of invertebrates and 
+                            small vertebrates, including the occasional small snake. Since being introduced in western Australia 
+                            and New Zealand, the kookaburra has angered farmers by preying on their fowl. 
+                            The laughing kookaburra has dark brown wing plumage and a white head and underside. 
+                            Dark brown eye stripes run across its face and its upper bill is black. 
+                            Its reddish-coloured tail is patterned with black bars.`,
+        length: "43cm",
+        weight: "300g",
+        found: "Australia wide",
+        groupPage: "html/birds.html",
+    },
+
+    yellowTailedBlackCockatoo: {
+        name: "Yellow-Tailed Black Cockatoo",
+        lifespan: "41 years",
+        group: "Birds",
+        image: "images/transparent-img/yellow-tailed-black-cockatoo.PNG",
+        food: "Fruit, seeds and other plant material",
+        shortDescription:   `It has a short crest on the top of its head. Its plumage is mostly brownish black and 
+                            it has prominent yellow cheek patches and a yellow tail band. The body feathers are edged 
+                            with yellow giving a scalloped appearance.`,
+        fulldescription:    `The adult male has a black beak and pinkish-red 
+                            eye-rings, and the female has a bone-coloured beak and grey eye-rings. In flight, yellow-tailed 
+                            black cockatoos flap deeply and slowly, with a peculiar heavy fluid motion. Their loud, 
+                            wailing calls carry for long distances. The whiteae is found south of Victoria to the East 
+                            of South Australia and is smaller in size. The yellow-tailed black cockatoo is found in 
+                            temperate forests and forested areas across south and central eastern Queensland to 
+                            southeastern South Australia, including a very small population persisting in the Eyre Peninsula.`,
+        length: "65cm",
+        weight: "900g",
+        found: "Southeastern Australia",
+        groupPage: "html/birds.html",
+    },
+};    
+
+let activeAnimal = null;
+
+function updateContent(animal = null) {
+    const welcomeMessage = document.getElementById("welcome-message");
```

Comment:
här kan du avsluta med _el eller El på variablen så nästa person som kommer in här vet att det är en Element variable istället att man tror det är en string med message. Så du kan byta dom variablerna dom har en element som referrens

Reply:
Dessa variabler kan vi ha utanför funktionen också, så vi inte behöver "leta" efter dom varje knapp tryck